### PR TITLE
Added section about hourly weather forecast attributes

### LINKED ISSUE #26188: OpenWeatherMap Forecast data (for hourly - 5 day/3 hour data)
### Feedback

As I understand it, new registrations on OpenWeatherMap require the "hourly" selection. That works fine with me. I'm hoping to get information about temperatures during the next 24-hour period. It looks like the payload from the API call should contain temperatures every 3 hours for the upcoming 5 days. I do not understand how to access this additional data. I can only see the current and the upcoming 3-hour forecast. Could this helpful information be added to the website?

### URL

https://www.home-assistant.io/integrations/openweathermap/

### Version

2023.2.3

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by dannytsang**
## Proposed change
Also added example screenshot showing list structure of the forecast.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #26188

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/openweathermap.markdown [hunk 1]
```diff
@@ -104,3 +104,9 @@ The time period these sensors use depends on the forecast mode selected when con
 | `forecast_wind_speed`                | Wind speed for the forecast's time period, metre/sec.                                                                                                          |
 
 Details about the API are available in the [OpenWeatherMap documentation](https://openweathermap.org/api).
+
+### Forecast Weather Conditions Intervals
+
+Periodic forecast information is stored in the `forecast` attribute on the entity. To access and use the information should be reserved for advanced users using [Templates](/docs/configuration/templating/).
```
Comment (frenck) [inline source/_integrations/openweathermap.markdown]: This is a feature of the weather entity, should it be documented there?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/openweathermap.markdown [hunk 2]
```diff
@@ -18,7 +18,7 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The OpenWeatherMap weather integrations uses [OpenWeatherMap](https://openweathermap.org/) as a source for current meteorological data for your location.
+The OpenWeatherMap weather integrations uses [OpenWeatherMap](https://openweathermap.org/) as a source for current meteorological data for your location. It uses the [Weather integration platform](/integrations/weather/).
```
Comment (frenck) [inline source/_integrations/openweathermap.markdown]: That is an odd sentence to write, we don't write such things anywhere. Additionally, it uses more entity components, not just the `weather` one.

```suggestion
The OpenWeatherMap weather integrations uses [OpenWeatherMap](https://openweathermap.org/) as a source for current meteorological data for your location.
```

File: source/_integrations/weather.markdown [hunk 3]
```diff
@@ -34,3 +34,9 @@ The `weather` platform only knows the below listed conditions. The reason for th
 - 'windy'
 - 'windy-variant'
 - 'exceptional'
+
+### Forecast Weather Conditions Intervals
```
Comment (frenck) [inline source/_integrations/weather.markdown]:
```suggestion
## Forecast information
```

File: source/_integrations/weather.markdown [hunk 4]
```diff
@@ -34,3 +34,9 @@ The `weather` platform only knows the below listed conditions. The reason for th
...
+
+Periodic forecast information is stored in the `forecast` attribute on the entity. To access and use the information should be reserved for advanced users using [Templates](/docs/configuration/templating/).
+
+![Example OpenWeatherMap forecast attributes](/images/integrations/openweathermap/OpenWeatherMap_Attributes.png)
```
Comment (frenck) [inline source/_integrations/weather.markdown]: Can we inline a bit of a state YAML as an example instead? Screenshots are easy to get outdated.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/openweathermap.markdown (hunk 2 above)
Comment (dannytsang) [reply source/_integrations/openweathermap.markdown]: I will rethink this. I wanted a way to link back to the weather integration so people can see the common information like the forecast details.

File: source/_integrations/weather.markdown (hunk 4 above)
Comment (dannytsang) [reply source/_integrations/weather.markdown]: Good @frenck I will do that.

File: source/_integrations/openweathermap.markdown (hunk 2 above)
Comment (frenck) [reply source/_integrations/openweathermap.markdown]: Inntegration can either list information on what they provide, some add chapters on their sensors and other entities.

A more general solution would be leveraging the `ha_platforms` frontend matter, to automatically list them in the aside maybe... (but would consider that out of scope for this PR).

File: source/_integrations/weather.markdown (hunk 3 above)
Comment (dannytsang) [reply source/_integrations/weather.markdown]: Much better title. Updated in 3e25e8a57560e47829d5779f21f9806477a3674a

File: source/_integrations/weather.markdown (hunk 4 above)
Comment (dannytsang) [reply source/_integrations/weather.markdown]: Replaced image with YAML example. 3e25e8a57560e47829d5779f21f9806477a3674a

**Review by frenck — APPROVED** [review]
Thanks, @dannytsang 👍 

../Frenck

**Pull request merged**